app/services/credit_notes/create_from_upgrade.rb

```diff
@@ -27,6 +27,7 @@ def call
           },
         ],
         reason: :order_change,
+        automatic: true
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)